runtime/near-vm-logic/src/logic.rs

```diff
@@ -51,11 +50,6 @@ pub struct VMLogic<'a> {
     /// host-guest boundary.
     registers: HashMap<u64, Vec<u8>>,
 
-    /// Iterators that were created and can still be used.
```

Comment:
How is it related to the PR? Just a cleanup? I am not familiar with the module, so I don't know if it is ok to remove this code, but it seems that it was not used after all.

Reply:
We've removed iterators from the VMLogic, but didn't cleanup the code. It affects some functions from `TrieUpdate`. So removing iterator code helps.

Reply:
Fixes #2276